frontend/ui/src/services/types/sourceTypes.ts

```diff
@@ -0,0 +1,10 @@
+import {Source} from 'model';
+
+export const getSourceType = (source: string): Source => {
```

Comment:
I think you can simply access the values of an enum like so `Source[source]`, but I'm wondering what this even gets us? is it so that we get the `Source` enum type? In that case, this is the same as just casting, right?

Reply:
good catch! yes, `Source[source] `gives the right value 👍